packages/wrangler/src/config/validation-helpers.ts

```diff
@@ -434,6 +434,44 @@ export const validateRequiredProperty = (
 	return true;
 };
 
+/**
+ * Validate that at least one of the properties in the list is required.
+ */
+export const validateAtLeastOnePropertyRequired = (
+	diagnostics: Diagnostics,
+	container: string,
+	properties: {
+		key: string;
+		value: unknown;
+		type: TypeofType;
+	}[]
+): boolean => {
+	if (container) {
```

Comment:
We'd like to generally avoid mutating function arguments as it makes changes more difficult to track and test. Could this be refactored so that the `container` variable used inside this function has a different name? e.g.

```
const containerPath = container ? `${container}.` : '';
```